backend/tournesol/suggestions/suggested_video.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+from functools import total_ordering
+from typing import Optional
+
+
+@total_ordering
+class SuggestedVideo:
+    uid = ""
+    video1_score: float = 0
+    beta: dict[SuggestedVideo, float] = {}
+    nb_comparison_with: dict[str, int] = {}
+    global_video_score_uncertainty: float
+    global_video_score: float
+    suggestibility_normalization: float
+    NEW_NODE_CONNECTION_SCORE = 0.5
+
+    user_pref: float = 0
+
+    def __init__(
+            self,
+            from_uid: Optional[str] = None
+    ):
+        if from_uid is not None:
+            self.uid = from_uid
+        self.suggestibility_normalization = 1
+
+    def __eq__(self, __o: SuggestedVideo) -> bool:
+        return __o.uid == self.uid
+
+    def __le__(self, other: SuggestedVideo):
+        return self.uid <= other.uid
+
+    def __repr__(self) -> str:
+        return "Video " + self.uid
+
+    def __hash__(self):
+        return self.uid.__hash__()
+
+    def score_computation(self, reference: SuggestedVideo):
+        return (self.score_uncertainty + reference.score_uncertainty / (
+                    self.score + reference.score + 1)
+                ) / self.suggestibility_normalization
```

Comment:
Here `score_uncertainty`, `reference.score_uncertainty` may be uninitialized if no `EntityCriteriaScore` exists for the video yet. 
Should be define a default score here, or ignore the video before sorting by score?

Reply:
Added a default score, sounds more logical for the meaning of this variable